### PR TITLE
 ATLAS : Fix check_site_status : Closes #103

### LINKED ISSUE #103: ATLAS : Fix check_site_status
After the introduction of https://github.com/rucio/rucio/issues/5664 , the probe `check_site_status` needs to be updated to 1.29.0 to use `availability_read`, `availability_write` and `availability_delete` instead of `availability`. For this the probe needs to be updated to API + python3.

### PR DESCRIPTION
**Pull request by cserf**

### CODE REVIEW

File: atlas/check_site_status [hunk 1]
```diff
@@ -79,7 +76,7 @@ if __name__ == '__main__':
     RESP = requests.get(url=URL, verify='/etc/pki/tls/certs/ca-bundle.crt')
     DATA = json.loads(RESP.content)
     CRIC_RSES = []
-    for rse in DATA.values():
+    for rse in list(DATA.values()):
```
Comment (dchristidis) [inline atlas/check_site_status:79]: Why is this necessary?

Comment (cserf) [reply atlas/check_site_status]: Not necessary. Removed

**Review by dchristidis — APPROVED** [review]
